tools/ui-components/src/button.types.ts

```diff
@@ -3,8 +3,8 @@ type ButtonSize = 'small' | 'medium' | 'large';
 export interface ButtonProps {
   primary?: boolean;
   size?: ButtonSize;
-  backgroundColor?: string;
```

Comment:
No longer needed since the color change logic will depend inside the button and will depend on the other props.